### DISCUSSION TITLE
Zion: a possible Nostr rival

### DISCUSSION THREAD

**Discussion by serrq** [KoalaSat/nostros#341] [General]
https://www.zion.fyi/

https://guide.getzion.com/

What do you think about it?

**Comment by KoalaSat** [comment]
Nostros and many other nostr clients are FOSS (Free and Open Source Software) and anything under that concept doesn't have "rivals", since there is literally nothing to win or lose. 

With that being said, for-profit organizations trying to run their business model are really welcome to nostr, as far they recpect the protocol.
